lldb/test/API/functionalities/breakpoint/serialize/TestBreakpointSerialization.py

```diff
@@ -49,6 +49,37 @@ def test_scripted_extra_args(self):
         self.setup_targets_and_cleanup()
         self.do_check_extra_args()
 
+    def test_resolver_serialization(self):
+        """Test that breakpoint resolvers contain the expected information"""
+        self.build()
+        self.setup_targets_and_cleanup()
+
+        sym_ctx_list = self.orig_target.FindFunctions("main")
```

Comment:
If you want to be really safe you can call FindFunctions on the main module only:
```
sym_ctx_list = self.orig_target.module[0].FindFunctions("main")
```
This will ensure we never find more than one 'main'.

Reply:
Can we guarantee that the first module is always the binary we're debugging? This feels like an implicit assumption.

Reply:
yes, this will always work, but if it makes you feel better you can do:
```
exe_path = self.getBuildArtifact("a.out")
exe_module = self.orig_target.module[exe_path]
self.assertTrue(exe_module.IsValid(), 'Make sure we found the executable module')
sym_ctx_list = exe_module.FindFunctions("main")
```